jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -130,7 +129,7 @@
      */
     @JDIAction("Assert that '{name}' is hidden")
     public DataListAssert<T, D> hidden() {
-        jdiAssert(map(element, this::isDisplayed), everyItem(Matchers.is(false)));
+        jdiAssert(map(element(), this::isDisplayed), everyItem(Matchers.is(false)));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5576697)